CONTRIBUTING.md

```diff
@@ -1,108 +1,117 @@
 # How to Contribute
- We welcome all contributions, suggestions, and feedback, so please do not hesitate to reach out!
 
- 
- We'd love to accept your patches and contributions to this project. There are just a few small guidelines you need to follow.
+We welcome all contributions, suggestions, and feedback, so please do not hesitate to reach out!
+
+We'd love to accept your patches and contributions to this project. There are just a few small guidelines you need to follow.
 
 ## Contributor License Agreement
 
- Contributions to this project must be accompanied by a Contributor License
- Agreement. You (or your employer) retain the copyright to your contribution;
- this simply gives us permission to use and redistribute your contributions as
- part of the project. Head over to <https://cla.developers.google.com/> to see your current agreements on file or to sign a new one.
+Contributions to this project must be accompanied by a Contributor License
+Agreement. You (or your employer) retain the copyright to your contribution;
```

Comment:
Is this statement accurate? `You (or your employer) retain the copyright to your contribution;` 

My understanding is that the copyright belongs to "The Kyverno Authors" and not the contributor or their employer.